source-academy/cadet-frontend
# Fixed achievement-related issues

### Description
Resolves #1752 by adding a condition in processEvent().
Resolves #1439 by using moment.js in the achievement DateHelper to prettify dates.

### Type of change


### How to test
Run code while not logged in, no "Please log in again" pop out should appear.
Set achievements with different deadlines, see that the deadline listed in the achievement dashboard looks reasonable. 

### Checklist
